modules/core/src/main/scala/gem/math/Offset.scala

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2016-2017 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package gem
+package math
+
+final case class Offset(p: Offset.P, q: Offset.Q)
+object Offset {
+
+  val Zero: Offset =
+    Offset(P.Zero, Q.Zero)
+
+  final case class P(toAngle: Angle)
+  object P {
+    val Zero: P =
+      P(Angle.Angle0)
+  }
+
+  final case class Q(toAngle: Angle)
+  object Q {
+    val Zero: Q =
+      Q(Angle.Angle0)
+  }
```

Comment:
Will we define `Order`, etc. or the equivalent in cats once we switch?  I guess you did say you would build it out more in a subsequent PR.